deploy/gk-deploy

```diff
@@ -476,10 +476,14 @@ fi
 
 if [[ ${LOAD} -eq 0 ]]; then
   if [[ "${CLI}" == *oc\ * ]]; then
-    eval_output "${CLI} process -p HEKETI_ADMIN_KEY=${ADMIN_KEY} -p HEKETI_USER_KEY=${USER_KEY} deploy-heketi | ${CLI} create -f - 2>&1"
+    oc_version = $(${CLI} version | grep "oc")
+    if [[ "${oc_version}" == *3.4.* ]]; then
```

Comment:
Right, i *think* the current deployment scheme works for versions >= 3.4 / 1.4 only anyways, so it's probably not dangerous?

We might need to add additional checks when openshift deviates from the 3.X.Y / 1.X.Y pattern...

@humblec I did the work of updating the patch for you, see PR #244.